app/helpers/application_helper.rb

```diff
@@ -72,4 +72,20 @@ def content_tag_if(condition, *args, &block)
       capture(&block)
     end
   end
+
+  def admin_loans_path(*args)
+    # Default to active loans, unless overriden
+    options = { loans: { f: { status_value: ['active'] } } }
+    options = args.pop if args.last.is_a? Hash
```

Comment:
I guess that depends on what we want the behavior to be.

Right now there is just one link that uses options:

<img width="765" alt="red_emma_s__llc_-_madeline" src="https://cloud.githubusercontent.com/assets/59398/22573265/5921a116-e965-11e6-986b-08138acc447b.png">

With the change, it will show take you to loans, but default to only active loans. If we want to show all loans, we run into a similar problem with wice_grid where you can't choose to show all statues.